Formula/cppcheck.rb

```diff
@@ -21,6 +21,10 @@ class Cppcheck < Formula
 
   needs :cxx11
 
+  def addons_dir # Install the cppcheck python addons in this directory
```

Comment:
Make this a variable or just inline it rather than a function. Also, feels like this should be `#{pkgshare}/addons` or something